themes/socialbase/templates/system/html.html.twig

```diff
@@ -29,6 +29,7 @@
     not root_path ? 'path-frontpage' : 'path-' ~ root_path|clean_class,
     node_type ? 'page-node-type-' ~ node_type|clean_class,
     node_edit ? 'page-node-edit',
+    style == 'sky' ? 'socialblue--sky' : '',
```

Comment:
We should add this in an attributes object instead because the `style` variable is not part of socialbase.